From 19eb7894b8b3ac7eec3c6b346e17d250fe07a540 Mon Sep 17 00:00:00 2001
From: Denys Vlasenko <vda.linux@googlemail.com>
Date: Wed, 4 Apr 2012 11:07:10 +0200
Subject: [PATCH 12/53] debuginfo.py: fix abort handling

Hopefully helps with this:

Coredump references 132 debuginfo files, 101 of them are not installed
Setting up yum repositories
      <I press [stop] button in GUI>
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/urlgrabber/grabber.py", line 1688, in _progress_update
    def _progress_update(self, download_total, downloaded, upload_total, uploaded):
  File "/usr/bin/abrt-action-install-debuginfo", line 45, in sigterm_handler
    clean_up()
  File "/usr/lib/python2.7/site-packages/reportclient/debuginfo.py", line 85, in clean_up
    if ex.errno != errno.ENOENT:
NameError: global name 'errno' is not defined

Signed-off-by: Denys Vlasenko <vda.linux@googlemail.com>
---
 src/client-python/debuginfo.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/client-python/debuginfo.py b/src/client-python/debuginfo.py
index 34f9d3f..144f451 100644
--- a/src/client-python/debuginfo.py
+++ b/src/client-python/debuginfo.py
@@ -6,6 +6,7 @@ from reportclient import _
 import sys
 import os
 import time
+import errno
 
 old_stdout = -1
 def mute_stdout():
-- 
1.7.11.2

